<?xml version="1.0" encoding="UTF-8"?>
<launch>
	<param name="max_speed" value="1.0" />
	<param name="topic_keyboard" value="/KeyboardPub"/>

	<!-- Run nodes -->
	<node pkg="course_2023_pkg" name="teleop_robot" type="teleop_robot.py"/>

</launch>

